quail/utils.py

```diff
@@ -33,10 +33,16 @@
 
 
 def collect_literal_inputs(request):
+    csvs = [
+        "tmax_file_content",
+        "tmin_file_content",
+        "prec_file_content",
+        "tavg_file_content",
+    ]
```

Comment:
A move elegant way of doing this may be to include a `str` check in the list comprehension below.
```
# something like this:
if "data_type" in vars(request.inputs[k][0]).keys() and "_content" not in k
```